src/FormTaxonomies/resources/form-builder/form-tags.tsx

```diff
@@ -0,0 +1,60 @@
+import {__} from "@wordpress/i18n";
+import {debounce} from "@wordpress/compose";
+import apiFetch from '@wordpress/api-fetch';
+import {FormTokenField} from "@wordpress/components";
+import {getInitialFormTags} from "./windowData";
+
+/**
+ * @note Not sure why it won't let me use a state hook here.
+ *       As a workaround, I'm overloading the form settings.
+ *       They will be parsed out by the server during publish.
```

Comment:
Can you explain what you were trying to accomplish here?

Reply:
I had to overload the form settings to persist the search results, because I'm getting errors when using React hooks, like `useState()`.

Reply:
Resolving as moot.